src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncPivotBlockSelector.scala

```diff
@@ -0,0 +1,204 @@
+package io.iohk.ethereum.blockchain.sync
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Cancellable, Props, Scheduler}
+import akka.util.ByteString
+import io.iohk.ethereum.domain.BlockHeader
+import io.iohk.ethereum.network.EtcPeerManagerActor.PeerInfo
+import io.iohk.ethereum.network.PeerEventBusActor.PeerEvent.MessageFromPeer
+import io.iohk.ethereum.network.PeerEventBusActor.SubscriptionClassifier.MessageClassifier
+import io.iohk.ethereum.network.PeerEventBusActor.{PeerSelector, Subscribe, Unsubscribe}
+import io.iohk.ethereum.network.p2p.messages.PV62.{BlockHeaders, GetBlockHeaders}
+import io.iohk.ethereum.network.{EtcPeerManagerActor, PeerId}
+import io.iohk.ethereum.utils.Config.SyncConfig
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.FiniteDuration
+
+class FastSyncPivotBlockSelector(
+    val etcPeerManager: ActorRef,
+    val peerEventBus: ActorRef,
+    val syncConfig: SyncConfig,
+    val scheduler: Scheduler
+) extends Actor
+    with ActorLogging
+    with PeerListSupport
```

Comment:
`PeersClient` instead?

Reply:
Why? We don't need PeersClient functionality here